daemons/compute/Readme.md

```diff
@@ -151,20 +180,51 @@ requests that match both `namespace` and `workflow`.
 | ----- | ---- | ----------- |
 | `uids` | string[] | List of data movement requests associated with the given workflow and namespace. |
 
+## Canceling a data movement request
+
+Data movement requests can be cancelled after creation of the request.
```

Comment:
Ensure you include the ask from Marty: Mention if the Cancel Request is synchronous or async

Reply:
Updated. @mcfadden8 could you take a look as well?